simba/optimizer_util.py

```diff
@@ -592,7 +608,7 @@ def charging_curve_to_soc_over_time(
         socs.append(soc)
         return np.array((times, socs)).T
 
-    while soc < args.desired_soc_opps:
+    while soc < final_value:
```

Comment:
would it make sense to give it an EPS? `while soc < final_value - eps)`

Reply:
I dont think its needed since Line 623ff catches situations where issues with very small powers can arise. 
A loop condition with eps lead to "ugly" numeric_curves which end at 0.99999 instead of 1. Result wise it would not matter I think